modules/related-posts/jetpack-related-posts.php

```diff
@@ -674,6 +674,14 @@ protected function _get_related_post_data_for_post( $post_id, $position, $origin
 			'title' => $this->_to_utf8( $this->_get_title( $post->post_title, $post->post_content ) ),
 			'date' => get_the_date( '', $post->ID ),
 			'format' => get_post_format( $post->ID ),
+			/**
+			 * Filters the rel attribute for the Related Posts' links.
+			 *
+			 * @since 3.7.0
+			 *
+			 * @param bool true Should the link rel attribute for Related Posts' links be dislpayed? Default is yes (true).
+			 */
+			'rel' => apply_filters( 'jetpack_relatedposts_filter_post_link_rel_enabled', true ),
```

Comment:
I would also consider passing the `$post_id` as a third arg; we do it in a lot of other filters in the related posts module and would allow more complicated filtering based on what post is returned.